packages/debug/index.js

```diff
@@ -25,6 +25,12 @@ module.exports = Object.assign({}, addonBaseConfig, {
     const options = this.getEmberDataConfig();
     const env = process.env.EMBER_ENV;
 
+    const parentIsEmberDataAddon = this.parent.pkg.name === 'ember-data';
+
+    if (options.includeDataAdapterInProduction === undefined) {
+      options.includeDataAdapterInProduction = parentIsEmberDataAddon;
+    }
```

Comment:
~~This seems vaguely unrelated to deprecation stripping (possibly a bug fix on its own merits), why is it included in this PR?~~

I see, this was moved.